base/8/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:8.9.4
+FROM node:8.10.0
```

Comment:
Why not use `node:8.10`? If it's a long term support version, you want to include the latest patches

Reply:
good point, we should just go with `8.10`